onboarding/web-infra.md

```diff
@@ -0,0 +1,23 @@
+# Web Infrastructure Onboarding
+
+This document is an outline of the tasks necessary to onboard a new member of the Web Infrastructure team.
+
+Please note these tasks should only be performed after the person's nomination has passed.
+
+## Tasks
+
+Tasks prefixed by `Member:` are for the new member to complete.
+Tasks prefixed by `Web Admin:` are for a member of `@nodejs/web-admins` to complete.
+
+- [ ] Web Admin: Create an issue in [nodejs/web-team](https://github.com/nodejs/web-team) specifically for onboarding the member and include this list in it.
+  - Please link any additional issues or PRs relating to the member's onboarding to this issue for record keeping.
+- [ ] Web Admin: Recommend member have some form of physical 2FA (i.e. Yubikey) or passkey enabled on their GitHub account, and that any SSH or GPG key attached to their GitHub account have passphrases.
+- [ ] Web Admin: Recommend member sign their Git commits if not already doing so.
```

Comment:
I feel like the "Web Admin: Recommend" aren't really needed, and should be put as a separate section "# Recommendations for New Members" of this file?